src/main/kotlin/com/cognifide/gradle/aem/instance/action/AwaitAction.kt

```diff
@@ -148,11 +159,11 @@ open class AwaitAction(project: Project, val instances: List<Instance>) : Abstra
                     if (init) {
                         if (response.statusLine.statusCode == HttpStatus.SC_UNAUTHORIZED) {
                             if (sync.basicUser == Instance.USER_DEFAULT) {
-                                logger.info("Switching instance credentials from defaults to customized.")
+                                logger.debug("Switching instance credentials from defaults to customized.")
                                 sync.basicUser = instance.user
                                 sync.basicPassword = instance.password
                             } else {
-                                logger.info("Switching instance credentials from customized to defaults.")
+                                logger.debug("Switching instance credentials from customized to defaults.")
                                 sync.basicUser = Instance.USER_DEFAULT
                                 sync.basicPassword = Instance.PASSWORD_DEFAULT
                             }
```

Comment:
Too much nested lambdas, it's hard to read.

Reply:
`progressFor`, `progressIndicator`, `progressTicks` can be converted to signle expresions functions